### PR TITLE
Test db

### PR DESCRIPTION
**Pull request by ary-odoo**

### CODE REVIEW

File: fdmc/models/pos.py [hunk 1]
```diff
@@ -105,14 +107,71 @@ def _create_account_move(self):
...
+        split_cash_receivable_lines = data.get('split_cash_receivable_lines')
+        combine_cash_receivable_lines = data.get('combine_cash_receivable_lines')
+        order_account_move_receivable_lines = data.get('order_account_move_receivable_lines')
+        invoice_receivable_lines = data.get('invoice_receivable_lines')
+        stock_output_lines = data.get('stock_output_lines')
+
+        for statement in self.statement_ids:
+            if not self.config_id.cash_control:
+                pass
+                # statement.write({'balance_end_real': statement.balance_end})
```
Comment (gaf-odoo) [inline fdmc/models/pos.py:131]: remove pls

Comment (ary-odoo) [reply fdmc/models/pos.py:131]: this comment is the way it's done in standard, I am keeping this here for reference in the future.

**Review by gaf-odoo — APPROVED** [review]
Approved, but remove the comment as to not increase LOC

**Comment by ary-odoo** [comment]
comments empty lines are not counted in LOC calculations, and this commented line in particular I need to get back to it in the future

**Pull request merged**